docs/BUILD.bazel

```diff
@@ -140,71 +139,12 @@ sphinx_build_binary(
     ],
 )
 
-_REQUIREMENTS_TARGET_COMPATIBLE_WITH = select({
-    "@platforms//os:linux": [],
-    "@platforms//os:macos": [],
-    "@platforms//os:windows": [],
-    "//conditions:default": ["@platforms//:incompatible"],
-}) if BZLMOD_ENABLED else ["@platforms//:incompatible"]
-
-# Run bazel run //docs:requirements.update
-genrule(
-    name = "requirements",
+# Run bazel run //docs:requirements_update
+pin(
```

Comment:
We're still keeping the api private right?

I'm not sold on the `pin` name. I think you were the one who actually convinced me the word should be `lock`. But happy to bikeshed names another time.

Reply:
I'll change it to `lock`, `pin` was just a placeholder, thanks for reminding!